### DISCUSSION TITLE
My First Preset VS My WIP Latest!

### DISCUSSION THREAD

**Discussion by Degamisu** [ntsc-rs/ntsc-rs#166] [General]
# My first preset - VHS1

I thought it would be a fun idea to look back at some of my old presets. I'm not sure when this was made but I'll figure it out soon.

I'm pretty sure I was just messing around with the settings to see what would happen, so this does **not** look like my best work 😆 

![image](https://github.com/user-attachments/assets/cefc6a25-4a12-408a-8f28-9ea48912fbc8)

# Sneak peak

And now for a sneak peak at my latest preset. Based off of a 1992 Fuji T-120 HG, it is probably one of my favorite ones I've created. It is multi use, so it can be used for video games, TV shows, and home videos that are centered, created, or staged in the date range of 1990 - 2001.

The tape preset has been in development for a while now, (around a month!) as I fine tune the settings. It is almost complete and should be released later today! Check back later or follow this discussion for the tape preset.

![image](https://github.com/user-attachments/assets/91142d56-497c-44a1-9919-b7a0b5631e16)
![image](https://github.com/user-attachments/assets/8d76f049-ecf6-4635-9316-236bd029fb84)
![image](https://github.com/user-attachments/assets/a8b2cb7c-1456-4f27-ba45-90ed02035bab)

All samples, including original images, were recorded by me.

**Comment by angryfrenchtoast** [comment]
can i have the new preset please

**Reply by Degamisu** [reply]
The preview version is available at #167 !

**Reply by angryfrenchtoast** [reply]
sorry i didnt see that, thanks!
